tensorflow-examples/src/main/java/org/tensorflow/model/examples/objectdetection/FasterRcnnInception.java

```diff
@@ -225,78 +258,71 @@ public static void main(String[] params) {
 
             DecodeJpeg.Options options = DecodeJpeg.channels(3L);
             DecodeJpeg decodeImage = tf.image.decodeJpeg(readFile.contents(), options);
+
             //fetch image from file
             TUint8 outputImage = (TUint8) runner.fetch(decodeImage).run().get(0);
             Shape imageShape = outputImage.shape();
+            //dimensions of test image
             long[] shapeArray = imageShape.asArray();
-
+            //reshape the tensor to 4D for input to model
+            Reshape<TUint8> reshape = tf.reshape(tf.constant(outputImage),
+                    tf.array(1,
+                            outputImage.shape().asArray()[0],
+                            outputImage.shape().asArray()[1],
+                            outputImage.shape().asArray()[2]
+                    )
+            );
+            TUint8 reshapeTensor = (TUint8) runner.fetch(reshape).run().get(1);
```

Comment:
You should probably use a fresh runner here, so it doesn't re-execute the decode.

Reply:
Will do